test/0.8/0.8-tests.js

```diff
@@ -338,4 +338,128 @@ describe('RAML 0.8 Tests', function () {
       chai.expect(stdoutLines[2]).to.contain(files[2]);
     });
   });
+
+  describe('Invalid file with one warning (default)', function () {
+
+    const ramlCop = path.join(__dirname, '..', '..', 'src', 'raml-cop.js');
+    const file    = path.join(__dirname, 'data', 'invalid-one-warning.raml');
+    
+    let results = {};
+      
+    before(function (done) {
+
+      testChildProcess(ramlCop, [file], (err, data) => {
+        if (err) { return done(err); }
+
+        results = data;
+        done();
+      });
+    });
+
+    it ('STDOUT should contain filename, line number, column number, and message for both error and warning', function() {
```

Comment:
The wrong line was the 359, here I test a file with one error and one warning. I just fix the previous line.